rcp/org.testeditor.rcp4.views.teststepselection/plugin.xml

```diff
@@ -4,10 +4,6 @@
    <extension
          id="org.testeditor.rcp4.views.teststepselector.fragment"
          point="org.eclipse.e4.workbench.model">
-      <fragment

```

Comment:
I just wanted to remove the refresh button. Now I got it. I will add the missing code again.